### PR TITLE
Scale converter

### PR DESCRIPTION
**Pull request by shaiTheKimhi**

### CODE REVIEW

File: Modules/Control/Scale_Convert.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+'''
+Input:
+    tilt - the tilt of the wanted scale in comparison to the given scale
+    vector - the given vector for the first scale
+Output:
+    the given vector for the 2nd scale
+'''
+def convert(tilt:numpy.ndarray ,vector:numpy.ndarray) ->numpy.ndarray:
+    #this function converts between different scales for a known tilt
+    x = vector[0]-tilt[0]
```
Comment (yotam180) [inline Modules/Control/Scale_Convert.py]: Operator spacing `vector[0]-tilt[0]` -> `vector[0] - tilt[0]`

File: Modules/Control/Scale_Convert.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import numpy
+
+'''
+Input:
+    tilt - the tilt of the wanted scale in comparison to the given scale
+    vector - the given vector for the first scale
+Output:
+    the given vector for the 2nd scale
+'''
+def convert(tilt:numpy.ndarray ,vector:numpy.ndarray) ->numpy.ndarray:
```
Comment (yotam180) [inline Modules/Control/Scale_Convert.py]: 1. Move the comma `tilt:numpy.ndarray ,vector:numpy.ndarray` -> `tilt:numpy.ndarray, vector:numpy.ndarray`
2. Operator spacing `) ->numpy.ndarray:` -> `) -> numpy.ndarray:`

File: Modules/Control/Scale_Convert.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    #this function converts between different scales for a known tilt
+    x = vector[0]-tilt[0]
+    y = vector[1] - tilt[1]
+    z = vector[2] - tilt[2]
+    return numpy.ndarray([x,y,z])
```
Comment (yotam180) [inline Modules/Control/Scale_Convert.py]: Comma spacing `numpy.ndarray([x,y,z])` -> `numpy.ndarray([x, y, z])`

**Review by yotam180 — COMMENTED** [review]
Just some **very very** little corrections I added in review comments :slightly_smiling_face:.

File: Modules/Control/Scale_Convert.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import numpy
+
+'''
+Input:
+    tilt - the tilt of the wanted scale in comparison to the given scale
+    vector - the given vector for the first scale
+Output:
+    the given vector for the 2nd scale
+'''
+def convert(tilt:numpy.ndarray ,vector:numpy.ndarray) -> numpy.ndarray:
```
Comment (yotam180) [inline Modules/Control/Scale_Convert.py]: Psik and then Revach

Translation for Shai:
פסיק ואז רווח

**Review by yotam180 — COMMENTED** [review]
Look at the tguva

**Review by KorenX — APPROVED** [review]
comma is finally in the right place; approved.

**Pull request merged**